mypy/nodes.py

```diff
@@ -731,6 +731,7 @@ class ClassDef(Statement):
     info = None  # type: TypeInfo  # Related TypeInfo
     metaclass = ''  # type: Optional[str]
     decorators = None  # type: List[Expression]
+    analyzed = None  # type: Optional[Expression]
```

Comment:
I think we don't need to serialize this (since it is needed only for semantic analysis). I updated the comment.